Makefile

```diff
@@ -31,10 +31,10 @@ help:  ## Display this help
 
 .PHONY: gazelle
 gazelle: ## Run Bazel Gazelle
-	(which bazel && bazel run //:gazelle) || true
+	(which bazel && ./hack/update-bazel.sh) || true
 
 .PHONY: test
-test: generate fmt vet manifests verify ## Run tests
+test: verify generate fmt vet manifests ## Run tests
```

Comment:
I agree with this, otherwise it looks like our tests are not checking that the clients are correctly generated.  But a caveat: there's no guarantee on the execution order here by make (though we should be OK as long as we're not doing parallel makefile execution)